.env

```
@@ -87,4 +87,15 @@ KC_DB_OPTS="--db postgres --db-url jdbc:postgresql://kc-ssi-instance-1.clh0lvey1
 # For AWS: Remember to provide cert and key pem files ($WORK_DIR/../env/.env)
 KC_START="start --hostname-strict=false --https-port=$KEYCLOAK_HTTPS_PORT --https-certificate-file=$KC_SERVER_CERT --https-certificate-key-file=$KC_SERVER_KEY"
 # For local env
-# KC_START=start-dev
\ No newline at end of file
+# KC_START=start-dev
+
+# Keycloak config CLI
+REPO_URL="https://github.com/adorsys/keycloak-config-cli.git"
+KC_CLI_DIR=$(pwd)/config
```

Prefer reusing WORK_DIR.
```suggestion
KC_CLI_DIR=$WORK_DIR/config
```